Sources/PrimerSDK/Classes/Services/Network/PrimerAPIClient.swift

```diff
@@ -145,8 +146,13 @@ internal class PrimerAPIClient: PrimerAPIClientProtocol {
         }
     }
     
-    func exchangePaymentMethodToken(clientToken: DecodedJWTToken, paymentMethodId: String, completion: @escaping (_ result: Result<PrimerPaymentMethodTokenData, Error>) -> Void) {
-        let endpoint = PrimerAPI.exchangePaymentMethodToken(clientToken: clientToken, paymentMethodId: paymentMethodId)
+    func exchangePaymentMethodToken(
+        clientToken: DecodedJWTToken,
+        vaultedPaymentMethodId: String,
+        vaultedPaymentMethodAdditionalData: PrimerVaultedPaymentMethodAdditionalData?,
+        completion: @escaping (_ result: Result<PrimerPaymentMethodTokenData, Error>) -> Void
+    ) {
+        let endpoint = PrimerAPI.exchangePaymentMethodToken(clientToken: clientToken, vaultedPaymentMethodId: vaultedPaymentMethodId, vaultedPaymentMethodAdditionalData: vaultedPaymentMethodAdditionalData)
```

Comment:
Are you sure that the CVV cannot end up in a log or an error thrown in case of an issue?

Reply:
It will only appear in the console logs in `DEBUG` mode in the request's body